tools/Replay.hpp

```diff
@@ -152,6 +165,8 @@ struct Stats {
     std::unordered_map<std::string, MessageStat> _recvd;
     std::unordered_map<std::string, MessageStat> _sent;
 
+    std::vector<PerfMetricInfo> perfStatsList;
```

Comment:
Is that a member - if so it needs an _ prefix so code-readers can tell that -cf. all other member variables =)